erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -301,6 +301,9 @@ def make_payment_request(**args):
 
 	if existing_payment_request:
 		frappe.db.set_value("Payment Request", existing_payment_request, "grand_total", grand_total, update_modified=False)
+		frappe.db.set_value("Payment Request", existing_payment_request, "payment_gateway", gateway_account.payment_gateway, update_modified=False)
```

Comment:
I just saw the end of the line lol, this should be modified so that Frappe can show any changes that may happen.